examples/suspense/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "suspense",
+    "version": "1.0.0",
+    "main": "index.js",
+    "license": "MIT",
+    "dependencies": {
+      "isomorphic-unfetch": "3.0.0",
+      "next": "9.1.1",
+      "swr": "^0.1.9"
```

Comment:
You could use latest, without the lock file it will really point to the latest version of SWR